contentcuration/contentcuration/frontend/shared/views/Breadcrumbs.vue

```diff
@@ -99,7 +99,7 @@
         this.$nextTick(() => {
           if (this.$refs.breadcrumb) {
             for (var i = this.$refs.breadcrumb.length - 1; i >= 0; --i) {
-              totalWidth += this.$refs.breadcrumb[i].$el.offsetWidth + 40;
```

Comment:
Unfortunately, this still doesn't fix the issue. If anything, it would be a partial fix--in some screen sizes it works on others it doesn't